docs/application/toc_all.md

```diff
@@ -262,6 +262,7 @@
 ### [Create Samsung Smart TV Web Application](/application/web/get-started/tv/first-samsung-tv-app.md)
 ### [Create Mobile Web Application](/application/web/get-started/mobile/first-app.md)
 ### [Create Web Application Addon](/application/web/get-started/addon/first-addon.md)
+### [Create Web Service Application](/application/web/get-started/web-service/first-service.md)
```

Comment:
I'm sorry that I forgot the web service app issue a long time ago.
The Web service app is not supported for Wearable so some developers are confused.
The Web Service Application guide says that **this feature is supported in TV applications only.** in web/guides/applications/service-app.md.

However, all screenshots are based on mobile in this guide.

Reply:
The legacy Web service has been enabled only for TV profile but this new Web service is enabled for all profiles including wearable and mobile since Tizen 5.5. Please check the Web service on mobile and wearable emulator. This has mobile screenshot only because it's same as wearable.